packages/core/lib/settings/pubspec.g.dart

```diff
@@ -80,7 +80,7 @@ FlutterGen _$FlutterGenFromJson(Map json) => $checkedCreate(
         );
         final val = FlutterGen(
           output: $checkedConvert('output', (v) => v as String),
-          lineLength: $checkedConvert('line_length', (v) => v as int),
+          lineLength: $checkedConvert('line_length', (v) => (v as num).toInt()),
```

Comment:
@AlexV525 
It might be better to detect changes in this file.

Reply:
This should be caused by dependencies upgrade like `json_serializable`.